pkg/agent/openflow/pipeline.go

```diff
@@ -2659,7 +2659,7 @@ func (c *client) realizePipelines() {
 			tables[i].SetNext(nextID)
 			tables[i].SetMissAction(missAction)
 			// Realize the table on OVS bridge.
-			c.bridge.CreateTable(tables[i], nextID, missAction)
+			c.bridge.NewTable(tables[i], nextID, missAction)
```

Comment:
Is the comment still correct? Does it realize the table on OVS bridge?

Reply:
removed the comment.